Makefile

```diff
@@ -22,7 +22,7 @@ clean-all:
 	rm -rf ./node_modules ./packages/*/node_modules ./packages/*/{lib,esm}
 
 bootstrap: clean-all
-	yarn install
+	$(LERNA) bootstrap
```

Comment:
The CI is failing because `lerna` is not installed, I'm using the local executable provided during the `yarn install`. Either we install lerna manually before or leave it as is.

Reply:
You are right. Closing this.